manifests/alertmanager-alertmanager.yaml

```diff
@@ -1,35 +1 @@
-apiVersion: monitoring.coreos.com/v1
-kind: Alertmanager
-metadata:
-  labels:
-    alertmanager: main
-    app.kubernetes.io/component: alert-router
-    app.kubernetes.io/name: alertmanager
-    app.kubernetes.io/part-of: kube-prometheus
-    app.kubernetes.io/version: 0.22.2
-  name: main
-  namespace: monitoring
-spec:
-  image: quay.io/prometheus/alertmanager:v0.22.2
-  nodeSelector:
-    kubernetes.io/os: linux
-  podMetadata:
-    labels:
-      app.kubernetes.io/component: alert-router
-      app.kubernetes.io/name: alertmanager
-      app.kubernetes.io/part-of: kube-prometheus
-      app.kubernetes.io/version: 0.22.2
-  replicas: 3
-  resources:
-    limits:
-      cpu: 100m
-      memory: 100Mi
-    requests:
-      cpu: 4m
-      memory: 100Mi
-  securityContext:
-    fsGroup: 2000
-    runAsNonRoot: true
-    runAsUser: 1000
-  serviceAccountName: alertmanager-main
-  version: 0.22.2
+null
```

Comment:
This shouldn't be removed.

Reply:
strangly it happens when I run the build.sh. I also get the warning `cat: manifests/alertmanager-alertmanager: No such file or directory`

Reply:
Weird. Try with `make manifests` :)

Reply:
Seems to work now, thanks! 😄